pages/docs/quickstart.mdx

```diff
@@ -114,6 +117,7 @@ You can skip `optional extended message`
 ![propose new file](https://user-images.githubusercontent.com/82668196/208292186-d954f0e5-bdb0-4177-ac14-eb5d99b329dd.png)
 
 13. Click `compare across forks` and you will see 4 drop downs (**HIGHLY IMPORTANT - MANY GO WRONG HERE**)
+<Table className="buggyTable">
```

Comment:
Probably could use mdx custom elements here, it would be much more reusable
https://nextjs.org/docs/advanced-features/using-mdx#custom-elements

Reply:
I have gone through the Url its says
_When you want to style your own elements to give a custom feel to your website or application, you can pass in shortcodes_. 
what I understand is if I make changes targeting 1 table element writing its Css then it would affect all the tables on the website. So I choose to go with className than element name.